govc/README.md

```diff
@@ -170,6 +175,17 @@ To configure the debug output you can use two environment variables.
 
 The [debug-format](../scripts/debug-format.sh) script can be used to format the debug output similar to the `-trace` flag.
 
+### Print Version Information
+
+For troubleshooting and when filing issues, get build related details with:
+
+```bash
```

Comment:
I like using `console` for these, output is colorized.  Prefix commands with a prompt `% govc ...`, for example:
```console
% govc version -l
Build Version: v0.25.0-next
Build Commit: e86da96e
Build Date: 2021-04-19T10:29:57Z
```

Reply:
Oh, nice!!!!